### ISSUE TITLE
UI bug

### ISSUE THREAD

**Issue by ymohit1603** [myntra/ecommerce-demo#26]
Buttons disappears off the screen

![Screenshot (20)](https://github.com/myntra/ecommerce-demo/assets/114051505/f9105c4f-5202-4978-a779-22daa0221986)

**Comment by Chandradeep-Pra** [comment]
Hi , is this issue still open ?

**Comment by ymohit1603** [comment]
Yes @Chandradeep-Pra this issue is still open but the repository is inactive ,So it doesn't mean any sense to contribute here

**Comment by Chandradeep-Pra** [comment]
Ok , thank you @ymohit1603
